aptos-move/framework/aptos-framework/sources/resource_account.spec.move

```diff
@@ -155,8 +155,6 @@ spec aptos_framework::resource_account {
         let get = len(optional_auth_key) == 0;
         let account = global<account::Account>(source_addr);
 
-        requires source_addr != resource_addr;
```

Comment:
Removed this because an abstract `ensures` clause is added in `create_resource_address` above.